src/argus_htmx/templates/htmx/incidents/_base_incident_update_modal.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'htmx/_base_form_modal.html' %}
+
+{% block form_control %}
+  {% if action_type == "bulk-update" %}
+    hx-post="{% url endpoint|default:'htmx:incident-list' %}"
+    hx-include="[name='selected_incidents']"
+  {% else %}
+    method="post"
+    action="{{ endpoint|default:'htmx:incident-list' }}"
```

Comment:
This will not generate an url. Compare with line 5.

Reply:
Line 9, if endpoint is set, you will get the endpoint. If it is not set, you will get the string "'htmx:incident-list".